core/server/services/settings/yaml-parser.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const yaml = require('js-yaml'),
+    Promise = require('bluebird'),
+    debug = require('ghost-ignition').debug('services:settings:yaml-parser'),
+    common = require('../../lib/common');
+
+/**
+ * Takes a YAML file, parses it and returns a JSON Object
+ * @param {YAML} file the YAML file utf8 encoded
+ * @param {String} fileName the name of the file incl. extension
+ * @returns {Promise<Object>} parsed
+ */
+module.exports = function parseYaml(file, fileName) {
```

Comment:
why is this module async? 🤔

Reply:
I thought it's better to have it async. I can refactor that to be sync tho, as the loader itself is async already.

Reply:
Changed it to be synchronous 😊